src/Agent.Worker/TaskManager.cs

```diff
@@ -429,11 +442,21 @@ public string WorkingDirectory
         }
     }
 
-    public sealed class PowerShell3HandlerData : HandlerData
+    public sealed class NodeHandlerData : BaseNodeHandlerData
     {
         public override int Priority => 2;
     }
 
+    public sealed class Node10HandlerData : BaseNodeHandlerData
+    {
+        public override int Priority => 1;
```

Comment:
do we support task has both node and node10 handler?

Reply:
Currently, we do. The task runner will prefer a Node10 task.json declaration over Node, due to its higher priority.